runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -87,6 +87,9 @@
 	private static final Pattern BARROWS_PATTERN = Pattern.compile("Your Barrows chest count is: <col=ff0000>(\\d+)</col>");
 	private static final Pattern KILL_DURATION_PATTERN = Pattern.compile("Fight duration: <col=ff0000>[0-9:]+</col>. Personal best: ([0-9:]+)");
 	private static final Pattern NEW_PB_PATTERN = Pattern.compile("Fight duration: <col=ff0000>([0-9:]+)</col> \\(new personal best\\)");
+	private static final Pattern DUEL_ARENA_WINS_PATTERN = Pattern.compile("You \\w+( \\w+)?! You have (now )?won (\\d+) duels");
```

Comment:
This seems like a fairly wide regex considering both test strings start with `You won!` - what other combinations are there?

Reply:
`You were defeated! You have won x duels.` and `You won! You have now won x duels.`

It matches wins and losses so even if you lose it track know how many you have won, since it is separated in two messages. Also the first duel needs to parse both wins and losses.

Reply:
so like the message is different for wins/losses